<dec f='boost-1.85.0/libs/asio/include/boost/asio/detail/throw_error.hpp' l='28' type='void boost::asio::detail::do_throw_error(const boost::system::error_code &amp; err, const boost::source_location &amp; loc)'/>
<use f='boost-1.85.0/libs/asio/include/boost/asio/detail/throw_error.hpp' l='42' u='c' c='_ZN5boost4asio6detail11throw_errorERKNS_6system10error_codeERKNS_15source_locationE'/>
<def f='boost-1.85.0/libs/asio/include/boost/asio/detail/impl/throw_error.ipp' l='28' ll='34' type='void boost::asio::detail::do_throw_error(const boost::system::error_code &amp; err, const boost::source_location &amp; loc)'/>
